### DISCUSSION TITLE
Why does "session capture" only work if the session has GFX enabled

### DISCUSSION THREAD

**Discussion by Mr-LZG** [FreeRDP/FreeRDP#8983] [Q&A] [answered]
Why does "session capture" only work if the session has GFX enabled？

**Comment by akallabeth** [comment]
because it only captures `GFX` traffic

**Reply by Mr-LZG** [reply]
I mean why the session that does not support GFX cannot use "session capture", is there a technical limitation?

**Reply by akallabeth** [reply, marked as answer]
yes, it is time.
nobody bothered to implement it as since windows 8 everything is gfx (even windows 7 got a backport) ;)
